client/a8c-for-agencies/sections/marketplace/checkout/request-client-payment.tsx

```diff
@@ -194,15 +204,35 @@ function RequestClientPayment( { checkoutItems }: Props ) {
 
 			<NoticeSummary type="request-client-payment" />
 
-			<div className="checkout__aside-actions">
+			<div
+				className="checkout__aside-actions"
+				role="button"
+				tabIndex={ 0 }
+				onMouseEnter={ () => setShowVerifyAccountToolip( true ) }
+				onMouseLeave={ () => setShowVerifyAccountToolip( false ) }
+				onMouseDown={ () => setShowVerifyAccountToolip( false ) }
+				onTouchStart={ () => setShowVerifyAccountToolip( true ) }
+			>
 				<Button
+					ref={ ctaButtonRef }
 					primary
 					onClick={ handleRequestPayment }
-					disabled={ ! hasCompletedForm }
+					disabled={ ! hasCompletedForm || isUserUnverified }
 					busy={ isPending }
 				>
 					{ translate( 'Request payment from client' ) }
+					{ isUserUnverified && <Icon icon={ warning } /> }
 				</Button>
+
+				<Tooltip
+					context={ ctaButtonRef.current }
+					isVisible={ showVerifyAccountToolip && isUserUnverified }
+					position="bottom"
+				>
+					{ translate(
+						'Please verify your account email in order to begin referring products to clients.'
```

Comment:
Not a blocker, but it might be helpful for users to link to wordpress.com/me from here, where they can resend the email if needed.

Reply:
Good suggestion. I'll make the **account email** a link then 👍

Reply:
Applied the suggestion. Thanks! 👍